parse-zoneinfo/Cargo.toml

```diff
@@ -13,3 +13,6 @@ keywords = ["date", "time", "timezone", "zone", "calendar"]
 version = "1.3.1"
 default-features = false
 features = ["std", "unicode-perl"]
+
+[dev-dependencies]
+insta = "1.38"
```

Comment:
I understand why you added this test. Not sure about it though.

Would it be better to add this test as a separate crate in the workspace?